lib/util/yaml.ts

```diff
@@ -166,9 +166,9 @@ export function dump(obj: any, opts?: DumpOptions): string {
   return stringify(obj, opts);
 }
 
-function massageContent(content: string, options?: YamlOptions): string {
+function messageContent(content: string, options?: YamlOptions): string {
   if (options?.removeTemplates) {
-    return stripTemplates(content.replace(regEx(/\s+{{.+?}}:.+/gs), ''));
+    return stripTemplates(content.replace(regEx(/^\s+{{.+?}}:.+/gs), ''));
```

Comment:
This isn't loosening?

Reply:
I will add my inputs to this discussion: https://github.com/renovatebot/renovate/discussions/32967